src/main/java/uk/gov/pay/publicauth/service/TokenService.java

```diff
@@ -56,7 +57,7 @@ private TokenHash encrypt(String token) {
     }
 
     private String createApiKey(String token) {
-        byte[] hmacBytes = HmacUtils.hmacSha1(apiKeyHmacSecret, token);
+        byte[] hmacBytes = new HmacUtils(HmacAlgorithms.HMAC_SHA_1, apiKeyHmacSecret).hmac(token);
```

Comment:
This is just inlining the contents of the deprecated `HmacUtils.hmacSha1(String, String)` method, which is what its Javadoc suggests